src/components/SearchBar.jsx

```diff
@@ -243,7 +243,7 @@ class SearchBar extends Component {
     }
 
     const theme = {
-      container: 'coz-searchbar-autosuggest-container' + (searching ? ' --searching' : ''),
+      container: 'coz-searchbar-autosuggest-container' + (searching ? ' --searching' : '') + (focused ? ' --focused' : ''),
```

Comment:
@CPatchane I don't think cozy-bar is using the `classname` module at all, would you mind if we added it to make this line cleaner?

Reply:
Yeah, sure :)